HOWTO.md

```diff
@@ -0,0 +1,135 @@
+#1. Overview
+In this document, we're going to focus on how to encode and decode OpenLR locations 
+on digital maps using modules in this package.
+
+#2. SetUp
+We'll need to add the following dependencies to our pom.xml
```

Comment:
Maybe just add the sentence.  Replace `${openlr.version}` with the version of the library you wish to use.